### PR TITLE
Add charge summary by type endpoint

### PR DESCRIPTION
**Pull request by AnnaGolosova**
## Link to JIRA ticket

https://app.clickup.com/t/1xd17wc

### *What is the problem we're trying to solve*

As a User, I would like to view the Asset apportionments

### *What changes have we introduced*

 Create a service level API endpoint to return Totals, Estate costs, Block costs, Property costs

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: FinanceServicesApi/V1/Boundary/Responses/PropertySummary/AssetAppointmentResponse.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+namespace FinanceServicesApi.V1.Boundary.Responses.PropertySummary
+{
+    public class AssetAppointmentResponse
```
Comment (sandipchandra) [inline FinanceServicesApi/V1/Boundary/Responses/PropertySummary/AssetAppointmentResponse.cs:6]: Its not appointment it should be apportionment, please make necessary changes wherever applicable.

**Review by sandipchandra — APPROVED** [review]
approved

**Review by sandipchandra — CHANGES_REQUESTED** [review]
Please make the necessary changes.

File: FinanceServicesApi/V1/Boundary/Responses/PropertySummary/AssetAppointmentResponse.cs (hunk 1 above)
Comment (AnnaGolosova) [reply FinanceServicesApi/V1/Boundary/Responses/PropertySummary/AssetAppointmentResponse.cs]: Renamed

**Review by FaisalHackney — APPROVED** [review]
The only thing I would add is some for the business logic to test that the figures for Apportionment of the Estate, Block and Property are as expected.

**Review by sandipchandra — APPROVED** [review]
approved

**Pull request merged**